src/components/Navbar/NavigationDrawer/NavigationDrawerBodySkeleton.tsx

```diff
@@ -0,0 +1,24 @@
+import Skeleton from '../../dls/Skeleton/Skeleton';
+
+import styles from './NavigationDrawerBodySkeleton.module.scss';
+
+const NAV_ROW_COUNT = 6;
+const rowsArr = Array(NAV_ROW_COUNT).fill(null);
+
+const NavigationDrawerBodySkeleton = () => {
+  return (
+    <span className={styles.skeletonContainer}>
+      {rowsArr.map((k, i) => (
+        // eslint-disable-next-line react/no-array-index-key
+        <Skeleton key={`skeleton_${i}`} isActive isSquared className={styles.navRow} />
+      ))}
+      <Skeleton isActive isSquared className={styles.blockRow} />
+      {rowsArr.map((k, i) => (
```

Comment:
2 things:
1. the `.map` code bit is duplicated so it makes sense to define it once and re-use it.
2. Because the code is running twice and the index is being used so for each element in the first loop there will be a corresponding element in the 2nd loop with the same key e.g. `skeleton_0` will exist twice, `skeleton_1` will exists twice and so on.